examples/gaussian.py

```diff
@@ -15,6 +15,7 @@
 model = A*exp(-((x - x0)**2/(2 * sig**2)))
 
 # Sample 10000 points from a N(15.0, 1.5) distrubution
+np.random.seed(seed=123456789)
```

Comment:
At least it works now - I think the bigger issue is symfit's sensitivity to inital guesses. See also #120, if scipy can do the job and symfit not something is wrong.